pkg/library/flatten/testdata/workspace-env/no-error-on-duplicate-workspace-env.yaml

```diff
@@ -0,0 +1,75 @@
+name: "Adds workspace environment variables from plugin"
```

Comment:
please allign test files names, now some are with _, some only -, some with CamelCase for workspaceEnv, some has workspace-env.

Reply:
Fixed for consistency -- note the format I've used thus far for `_` vs `-` is that dashes replace spaces in the sentence describing the test, and `error_` is the prefix that specifies that a test case is expected to return an error (it's mostly so that an alphabetical sort puts all error testcases together). If I've missed a place in that framework, I'll fix it.